cfme/common/vm.py

```diff
@@ -313,6 +313,7 @@ def find_quadicon(
         Returns: :py:class:`cfme.web_ui.Quadicon` instance
         Raises: VmOrInstanceNotFound
         """
+        from cfme.web_ui import paginator
```

Comment:
paginator class is implemented as module. 
When I add widgetastic paginator there, it gets initialized during import time, when neither appliance nor browser are ready.
So, I had to get rid of all global imports of paginator. 
Furthermore, I hope all these web_ui.paginator calls will be removed soon.